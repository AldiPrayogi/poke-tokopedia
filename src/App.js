import React from 'react';
import {HomePage} from "./Pages/HomePage";
import {Route, Switch, useHistory} from 'react-router-dom';
import {PokemonListPage} from "./Pages/PokemonListPage";
import {PokedexPage} from "./Pages/PokedexPage";
import {PokemonDetailPage} from "./Pages/PokemonDetailPage";

function App() {
    return(
      <Switch>
          <Route exact path='/' component={HomePage} />

          <Route path='/pokemon-list' component={PokemonListPage}/>

          <Route path='/pokedex' component={PokedexPage}/>

          <Route path='/pokemon-detail/:name' component={PokemonDetailPage} />
      </Switch>

    );
}

export default App;