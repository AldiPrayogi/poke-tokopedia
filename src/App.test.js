import App from './App';
import {shallow} from 'enzyme';

const wrapper = shallow(<App/>)

describe('App', () => {
  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
