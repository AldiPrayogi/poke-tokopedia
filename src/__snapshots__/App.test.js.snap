// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="/pokemon-list"
  />
  <Route
    component={[Function]}
    path="/pokedex"
  />
  <Route
    component={[Function]}
    path="/pokemon-detail/:name"
  />
</Switch>
`;
